rest_specification/restful_service_specification.md

```diff
@@ -161,7 +173,7 @@ corresponding to parameter names.
 Examples:
 
 ```json
-POST /user/findByLogin
+POST /user/v1/findByLogin
```

Comment:
I think we are agreeing on your first point. The only reason "v1" is in there is because of the versioning outlined earlier in the spec which says it is after the resource name in the path. It was just left out of the examples.

> What's the reasoning for making this a POST? Do we expect to send message bodies in order to query? I'd really prefer we don't allow that, but if we have to I can be swayed.

I'm not particular here, I could see additional functions having different HTTP methods, but @kahowell was leaning towards all POST for consistency and I could see that. Let's continue this discussion [here](https://github.com/lightblue-platform/lightblue-docs-specs/pull/43/files#r66153092).